projects/1.16-fabric/assets/ftb-chunks-fabric/ftbchunks/lang/zh_cn.json

```
@@ -0,0 +1,76 @@
+{
+  "ftbchunks": "FTB区块",
+  "ftbchunks.noise": "噪点",
+  "ftbchunks.shadows": "阴影",
+  "ftbchunks.chunk_grid": "区块网格",
+  "ftbchunks.reduced_color_palette": "简化色板",
+  "ftbchunks.saturation": "饱和度",
+  "ftbchunks.debug_info": "调试信息",
+  "ftbchunks.claimed_chunks_on_map": "在地图上显示认领的区块",
+  "ftbchunks.own_claimed_chunks_on_map": "在地图上显示你自己认领的区块",
+  "ftbchunks.in_world_waypoints": "在世界上显示路径点",
+  "ftbchunks.death_waypoints": "死亡时创建死亡路径点",
+  "ftbchunks.map_mode": "地图模式",
+  "ftbchunks.waypoint_fade_distance": "路径点淡出距离",
+  "ftbchunks.biome_blend": "生物群系过渡距离",
+  "ftbchunks.biome_blend.none": "无（最快）",
+  "ftbchunks.biome_blend.blend_3x3": "3x3",
+  "ftbchunks.biome_blend.blend_5x5": "5x5",
+  "ftbchunks.biome_blend.blend_7x7": "7x7",
+  "ftbchunks.biome_blend.blend_9x9": "9x9",
+  "ftbchunks.biome_blend.blend_11x11": "11x11",
+  "ftbchunks.biome_blend.blend_13x13": "13x13",
+  "ftbchunks.biome_blend.blend_15x15": "15x15（最慢）",
+  "ftbchunks.water_height_factor": "水面权重系数",
+  "ftbchunks.only_surface_entities": "仅地面实体",
+  "ftbchunks.experimental_performance_improvement": "实验性性能改进",
+  "ftbchunks.minimap": "小地图",
+  "ftbchunks.minimap.enabled": "启用",
+  "ftbchunks.minimap.position": "位置",
+  "ftbchunks.minimap.position.bottom_left": "左下",
+  "ftbchunks.minimap.position.left": "左",
+  "ftbchunks.minimap.position.top_left": "左上",
+  "ftbchunks.minimap.position.top_right": "右上",
+  "ftbchunks.minimap.position.right": "右",
+  "ftbchunks.minimap.position.bottom_right": "右下",
+  "ftbchunks.minimap.scale": "大小",
+  "ftbchunks.minimap.locked_north": "锁定北面",
+  "ftbchunks.minimap.waypoints": "路径点",
+  "ftbchunks.minimap.entities": "实体",
+  "ftbchunks.minimap.entity_heads": "实体图标",
+  "ftbchunks.minimap.player_heads": "玩家图标",
+  "ftbchunks.minimap.large_entities": "大型实体",
+  "ftbchunks.minimap.zoom": "缩放等级",
+  "ftbchunks.minimap.xyz": "XYZ",
+  "ftbchunks.minimap.biome": "生物群系",
+  "ftbchunks.minimap.blur_mode": "模糊模式",
+  "ftbchunks.minimap.blur_mode.auto": "自动",
+  "ftbchunks.minimap.blur_mode.on": "开",
+  "ftbchunks.minimap.blur_mode.off": "关",
+  "ftbchunks.minimap.compass": "指南针",
+  "ftbchunks.minimap.visibility": "可见度",
+  "ftbchunks.minimap.zone": "区域",
+  "sidebar_button.ftbchunks.chunks": "FTB区块",
+  "key.ftbchunks.map": "打开FTB区块地图",
+  "key.ftbchunks.minimap.zoomIn": "FTB区块放大",
+  "key.ftbchunks.minimap.zoomOut": "FTB区块缩小",
+  "wilderness": "野外",
+  "ftbchunks.no_server_mod": "服务器需要安装此模组！",
+  "ftbchunks.already_claimed": "此区块已被%s认领",
+  "ftbchunks.gui.claimed": "已认领",
+  "ftbchunks.gui.force_loaded": "强制加载",
+  "ftbchunks.gui.allies": "盟友",
+  "ftbchunks.gui.ally_whitelist": "盟友白名单",
+  "ftbchunks.gui.ally_blacklist": "盟友黑名单",
+  "ftbchunks.gui.large_map": "大地图",
+  "ftbchunks.gui.claimed_chunks": "认领区块",
+  "ftbchunks.gui.waypoints": "路径点",
+  "ftbchunks.gui.add_waypoint": "添加路径点",
+  "ftbchunks.gui.settings": "设置",
+  "ftbchunks.gui.sync": "与盟友共享地图",
+  "ftbteamsconfig.ftbchunks.allow_fake_players": "允许 Fake Players",

```

```suggestion
  "ftbteamsconfig.ftbchunks.allow_fake_players": "允许 Fake Player",
```
中文没有复数，所以夹杂在中文里的英文也应该没有复数？

---

无必要

---

遗憾的是《中文出版物夹用英文的编辑规范》[CY T154—2017](http://sxqx.alljournal.cn/uploadfile/sxqx/20190304/CY%20T154%E2%80%942017%20%E4%B8%AD%E6%96%87%E5%87%BA%E7%89%88%E7%89%A9%E5%A4%B9%E7%94%A8%E8%8B%B1%E6%96%87%E7%9A%84%E7%BC%96%E8%BE%91%E6%A0%87%E5%87%86.pdf)对此没有规定。